scripts/rollup/plugins/strip-unused-imports.js

```diff
@@ -11,8 +11,14 @@ module.exports = function sizes(pureExternalModules) {
     name: 'scripts/rollup/plugins/strip-unused-imports',
     transformBundle(code) {
       pureExternalModules.forEach(module => {
+        // Ideally this would use a negative lookbehind: (?<!= *)
+        // But this isn't supported by the version of Node that Prepack runs fasted on,
```

Comment:
"faster"? :-)

I think this clarification taken out of context is more confusing than it's worth.